test/test_no_multiply_numbers.py

```diff
@@ -0,0 +1,409 @@
+        category="to",
+        variants=["et", "þgf", "kk"],
+    )
+    check_terminal(
+        t[5],
+        text="milljarði",
+        lemma="milljarður",
+        category="no",
+        variants=["et", "þgf", "kk"],
+    )
+    check_terminal(
+        t[6],
+        text="króna",
+        lemma="króna",
+        category="no",
+        variants=["ft", "ef", "kvk"],
+    )
+
+    s = r.parse_single("Tjónið nam tuttugu og einum milljarði króna.")
+    assert s is not None
+    t = s.terminals or []
+    assert len(t) == 8
+    check_terminal(
+        t[2],
+        text="tuttugu",
+        lemma="tuttugu",
+        category="töl",
+        variants=[],
+    )
+    check_terminal(
+        t[4],
+        text="einum",
+        lemma="einn",
+        category="to",
+        variants=["et", "þgf", "kk"],
+    )
+    check_terminal(
+        t[5],
+        text="milljarði",
+        lemma="milljarður",
+        category="no",
+        variants=["et", "þgf", "kk"],
+    )
+    check_terminal(
+        t[6],
+        text="króna",
+        lemma="króna",
+        category="no",
+        variants=["ft", "ef", "kvk"],
+    )
+
+    s = r.parse_single("Fjöldi stjarna í Vetrarbrautinni skiptir hundruðum milljarða.")
+    assert s is not None
+    t = s.terminals or []
+    assert len(t) == 8
+    check_terminal(
+        t[5],
+        text="hundruðum",
+        lemma="hundrað",
+        category="no",
+        variants=["ft", "þgf", "hk"],
+    )
+    check_terminal(
+        t[6],
+        text="milljarða",
+        lemma="milljarður",
+        category="no",
+        variants=["ft", "ef", "kk"],
+    )
+
+    s = r.parse_single("Sex hundruð áttatíu og þrír leikmenn mættu á blakmótið.")
+    assert s is not None
+    t = s.terminals or []
+    assert len(t) == 10
+    check_terminal(
+        t[0],
+        text="Sex",
+        lemma="sex",
+        category="töl",
+        variants=[],
+    )
+    check_terminal(
+        t[1],
+        text="hundruð",
+        lemma="hundrað",
+        category="no",
+        variants=["ft", "hk", "nf"],
+    )
+    check_terminal(
+        t[2],
+        text="áttatíu",
+        lemma="áttatíu",
+        category="töl",
+        variants=[],
+    )
+    check_terminal(
+        t[3],
+        text="og",
+        lemma="og",
+        category="st",
+        variants=[],
+    )
+    check_terminal(
+        t[4],
+        text="þrír",
+        lemma="þrír",
+        category="to",
+        variants=["ft", "kk", "nf"],
+    )
+
+    s = r.parse_single("Tjónið nam tólf hundruðum punda.")
+    assert s is not None
+    t = s.terminals or []
+    assert len(t) == 6
+    check_terminal(
+        t[2],
+        text="tólf",
+        lemma="tólf",
+        category="töl",
+        variants=[],
+    )
+    check_terminal(
+        t[3],
+        text="hundruðum",
+        lemma="hundrað",
+        category="no",
+        variants=["ft", "þgf", "hk"],
+    )
+    # TODO: gets interpreted as 'pundur:kk'?
```

Comment:
We need an entry in the preferences.conf file for this, i.e. to always prefer the stem "pund" to the stem "pundur" for the word forms that are common to the two stems.

Reply:
Nice! This fixes it beautifully after I added the line to `Prefs.conf`.

Reply:
Weird, this fix doesn't work for the pypy3.6 tests on github. All the other tests pass though.

Reply:
I believe we agreed to drop Python 3.6 support some weeks back. Maybe you (@sultur) can update the docs in this repo to state that 3.7 is required, rm 3.6 tests in workflow and add pypy-3.7? Does that sound good to you, @vthorsteinsson?

Reply:
This change should probably also be documented in the release notes next time we release a new version of GreynirPackage on GitHub/PyPI.

Reply:
Yes, Python 3.7 as a minimum requirement is already decided, so we can safely drop the 3.6 stuff and change the documentation to that effect.

Reply:
The biggest change between 3.6 and 3.7 is that dicts are guaranteed to be ordered in the latter. Is this being relied on somewhere in the code?

Reply:
No, we should not have been relying on that, although our own servers run PyPy which has always guaranteed dicts to be in insertion order. But of course we can start relying on dict insertion order now, and the OrderedDicts we have in the code can be re-examined to determine whether they are still needed.